docs/relationship_synthesis.md

```diff
@@ -0,0 +1,252 @@
+    - Prometheus
+```
+
+One rule can be applied to multiple sources, as shown in the example. These origins are used by our internal services 
+to filter which rules to evaluate and which ones to ignore. 
+The mapping of origins to sources allows for performance improvements when evaluating rules via the matching system.
+
+Determining clear and specific values for defining origins is an ongoing discussion, 
+and further documentation will provide final proposals.
+
+## Conditions
+
+`Conditions` are used to determine if a rule should be applied to a given data point. 
+The proposal for `conditions` is an iterative improvement over the existing rules format to allow for 
+future changes without major breaking updates.
+
+The following example demonstrates various capabilities supported by conditions:
+
+```yaml
+# ...
+conditions:
+  - attribute: eventType
+    anyOf: ["NginxSample", "K8sPodSample", "K8sContainerSample"]
+  - attribute: host.guid
+    present: yes
+  - attribute: metricName
+    startsWith: "DurationByCaller/"
+    caseSensitive: true
+  - attribute: metricName
+    regex: ".*\.rds\.amazonaws\.com.*"
+# ...
+```
+
+The `attribute` field specifies the name of the attribute in the data point, and the operation to perform is defined using matchers 
+such as `anyOf`, `present`, `startsWith`, and `regex`. 
+Only one matcher should be present in each condition, and matching is case-insensitive by default unless 
+specified otherwise with `caseSensitive: true`. 
+Multiple conditions must all match for the rule to apply; there is no support for `OR` conditions.
+
+## Relationship
+
+The `relationship` section defines aspects related to the created relationship.
+
+### Expires
+
+The `expires` field allows configuring the duration for which the relationship should exist if it is not reported within that timeframe. 
+By default, this field is set to ***75 minutes*** (the default for the relationship platform). 
+The value for expires follows the `ISO-8601` format and must be between ***10 minutes*** and ***75 minutes***, 
+as per the platform documentation.
+
+Explicitly providing the expires field is recommended for clarity, even though the default is 75 minutes.
+
+### RelationshipType
+
+The `relationshipType` field specifies the type of relationship to create. Only predefined values from the 
+supported relationship types list are accepted.
+
+#### Source & Target Resolvers
+
+Resolvers define how the source and target GUIDs are determined. There are three types of resolvers:
+
+1. `extractGuid`: Extracts the GUID from an existing attribute in the telemetry.
+2. `buildGuid`: Constructs the GUID using various pieces of information from the telemetry.
+3. `lookupGuid`: Performs a lookup using candidates to find the matching GUID based on telemetry attributes. 
+
+##### extractGuid
+
+The `extractGuid` resolver is the simplest and allows specifying an attribute to use as the GUID.
+
+Example:
+
+```yaml
+# ...
+  source:
```

Comment:
nit: I would give an example with just `attribute` and do an extended version with the `entityType` ones (given these are more of a snowflake use cases)

Reply:
should be fixed now, please double check: https://github.com/newrelic/entity-definitions/pull/1138/commits/f530ab96a70339dd42672b2b4c9e17321cc0b7e4